README.md

```diff
@@ -119,6 +121,25 @@ This is an example of the value of `Specberus.meta` after the execution of `Spec
 }
 ```
 
+## REST API
+
+Similar to the [Node.js API](#js-api), Specberus exposes a REST API too.
+
+The endpoint is `<host>/api/`.
+
+There are three `GET` methods available:
+
+* `version`: returns the version string, eg `1.5.3`.
+* `metadata`: extract all known metadata from document; [returns JSON object with inferred properties](#extractmetadataoptions).
+* `validate`: check the document ([syntax](#validateoptions)); fails and returns an array of errors, or succeeds and returns the profile.
+
+Examples:
+
+* `<host>/api/version`
+* `<host>/api/metadata?url=http://example.com/doc.html`
+* `<host>/api/validate?file=/home/me/docs/spec.html`
```

Comment:
That method ('file' QS param) won't work unless the file is already on the server which won't happen. Same thing for 'document'.
Regarding 'source', it seems difficult to pass the whole source as the parameter.
I suggest to handle *only* 'url' for now.